src/components/Avatar.tsx

```diff
@@ -87,7 +87,7 @@ function Avatar({
 
     // We pass the color styles down to the SVG for the workspace and fallback avatar.
     const source = isWorkspace ? originalSource : UserUtils.getAvatar(originalSource, accountID);
-    const useFallBackAvatar = imageError || !source || source === Expensicons.FallbackAvatar;
+    const useFallBackAvatar = imageError || !originalSource || !source || source === Expensicons.FallbackAvatar;
```

Comment:
Revert. There shouldn't be a case where originalSource is empty and source is not. The bug is in `UserUtils.isDefaultAvatar`. Remove this part

https://github.com/Expensify/App/blob/95001698b2c3173f8c3d043458282e8e75bf90ce/src/libs/UserUtils.ts#L143-L146

Reply:
Done 👍  `UserUtils.isDefaultAvatar` is actually used in 2 more places, but I checked both and this change shouldn't break anything.
Uploading avatar displays correct options, and QR Share also works.